inc/cloudwatch_error_handler/namespace.php

```diff
@@ -48,6 +48,24 @@ function error_handler( int $errno, string $errstr, string $errfile = null, int
 	return false;
 }
 
+/**
+ * When script execution ends, send any buffered errors to CloudWatch.
+ */
+function send_buffered_errors_on_shutdown() {
+	$errors = $GLOBALS['altis_cloudwatch_error_handler_errors'];
+	if ( ! $errors ) {
+		return;
+	}
+
+	if ( function_exists( 'fastcgi_finish_request' ) ) {
+		fastcgi_finish_request();
+	}
+	send_buffered_errors();
+}
+
+/**
+ * Send all buffered errors to CloudWAtch.
+ */
 function send_buffered_errors() {
```

Comment:
Is this function not called anymore?

Reply:
Yup it's called right above, and still called from `error_handler` too.

Reply:
Ah, yeah, I missed the last line